pkgs/data/themes/breath/default.nix

```diff
@@ -0,0 +1,47 @@
+{ mkDerivation
+, lib
+, fetchFromGitLab
+, gitUpdater
+, cmake
+, extra-cmake-modules
+, kdecoration
+, plasma-workspace
+, qtbase
+}:
+
+mkDerivation rec {
+  pname = "breath-theme";
+  version = "3130eb1addeebb4b93f1192c377383618ebecaa3";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.manjaro.org";
+    owner = "themes";
+    group = "artwork";
+    repo = "breath";
+    rev = version;
+    sha256 = "1cc7zc31hhrzkcy6df1ckli2h4ij3lwclcm1wvcxsk24jdkr0hnb";
+  };
+
+  buildInputs = [
+    kdecoration
+    plasma-workspace
+    qtbase
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+  ];
+
+  cmakeFlags = [ "-DBUILD_PLASMA_THEMES=ON -DBUILD_SDDM_THEME=ON" ];
+
+  passthru.updateScript = gitUpdater { inherit pname version; };
```

Comment:
Can this work with `nix-update-script`? If yes, the `updateScript` is not required for @r-ryantm to work.

Reply:
Testing it with `passthru.updateScript = nix-update-script { attrPath = pname; };` and `nix-shell maintainers/scripts/update.nix --argstr package breath-theme --show-trace` seems to error as it can't find any git tags. `gitUpdater` also doesn't seem to work though.